modules/ROOT/pages/authentication-authorization/manage-users.adoc

```diff
@@ -743,33 +797,67 @@ The `SET PASSWORD` part is only optional if it directly follows the `SET PASSWOR
 * `SET HOME DATABASE` can be used to configure a home database for a user.
 A home database will be resolved if it is either pointing to a database or a database alias.
 If no home database is set, the DBMS default database is used as the home database for the user.
+[role=label--new-5.24]
+* One or more `SET AUTH` clauses can be used to set xref:authentication-authorization/auth-objects.adoc[Auth Objects], which define authentication / authorization providers for the user. This might be used to configure external auth providers like LDAP or OIDC, but can also be used as an alternative way to set the native (password-based) auth settings like `SET PASSWORD` and `SET PASSWORD CHANGE REQUIRED`. TODO: Insert links to the LDAP and OIDC examples.
 * `REMOVE HOME DATABASE` is used to unset the home database for a user.
 This results in the DBMS default database being used as the home database for the user.
+[role=label--new-5.24]
+* `REMOVE AUTH` is used to remove one or many xref:authentication-authorization/auth-objects.adoc[Auth Object(s)] from a user. It cannot be used such that a user would have no Auth Objects.
+* `REMOVE ALL AUTH` is used to remove all existing xref:authentication-authorization/auth-objects.adoc[Auth Objects] from a user and replace them with new ones. It must be used in conjunction with at least one `SET AUTH` clause.
 
-For example, you can modify the user `bob` with a new password and active status, and remove the requirement to change his password:
 
-[source, cypher, role=noplay]
+.Modify the user `bob` with a new password and active status, and remove the requirement to change his password:
+======
+[source,cypher,role=noplay]
 ----
 ALTER USER bob
 SET PASSWORD 'abcd5678' CHANGE NOT REQUIRED
 SET STATUS ACTIVE
 ----
 
-Or you may decide to assign the user `bob` a different home database:
+[role=label--new-5.24]
+The equivalent command using the xref:authentication-authorization/auth-objects.adoc[Auth Objects] syntax would be:
 
-[source, cypher, role=noplay]
+----
+ALTER USER bob
+SET AUTH 'native' {SET PASSWORD 'abcd5678' SET PASSWORD CHANGE NOT REQUIRED}
+SET STATUS ACTIVE
+----
+======
+
+.Modify the user `bob` to expire his current password so that he must change it the next time he logs in:
+======
+[source,cypher,role=noplay]
+----
+ALTER USER bob
+SET PASSWORD CHANGE REQUIRED
+----
+
+[role=label--new-5.24]
+The equivalent command using the xref:authentication-authorization/auth-objects.adoc[Auth Objects] syntax would be:
+
+----
+ALTER USER bob
+SET AUTH 'native' {SET PASSWORD CHANGE REQUIRED}
+----
+======
+
+.Assign the user `bob` a different home database:
+======
+[source,cypher,role=noplay]
 ----
 ALTER USER bob
 SET HOME DATABASE anotherDbOrAlias
 ----
 
-Or remove the home database from the user `bob`:
+Or remove the `HOME` database from the user `bob`:
```

Comment:
no, we usually refer to it as `home database` not ``` `HOME` database ```

Reply:
also this line didn't get the `. ======` refactoring but seems to have been added as part of the assign a new one example?

Reply:
ah yes, I had intended that it be part of the same example, but you're right, its better as its own example. Changed.